src/admin/components/ResourceTabs/PolygonReviewTab/index.tsx

```diff
@@ -0,0 +1,620 @@
+    openModal(
+      <ModalAdd
+        title="Add Polygons"
+        descriptionInput="Drag and drop a GeoJSON, Shapefile, or KML for your site Tannous/Brayton Road."
+        descriptionList={
+          <div className="mt-9 flex">
+            <Text variant="text-12-bold">TerraMatch upload limits:&nbsp;</Text>
+            <Text variant="text-12-light">50 MB per upload</Text>
+          </div>
+        }
+        onClose={closeModal}
+        content="Start by adding polygons to your site."
+        primaryButtonText="Save"
+        primaryButtonProps={{ className: "px-8 py-3", variant: "primary", onClick: () => setSaveFlags(true) }}
+        acceptedTYpes={FileType.ShapeFiles.split(",") as FileType[]}
+        setFile={setFiles}
+      >
+        {/* Next div is only Mocked data delete this children later*/}
+        {/* <div className="flex flex-col gap-4 mb-6">
+          {polygonData.map(polygon => (
+            <div
+              key={polygon.id}
+              className="border-grey-75 flex items-center justify-between rounded-lg border border-grey-750 py-[10px] pr-6 pl-4"
+            >
+              <div className="flex gap-3">
+                <div className="p-2 rounded-lg bg-neutral-150">
+                  <Icon name={IconNames.POLYGON} className="w-6 h-6 text-grey-720" />
+                </div>
+                <div>
+                  <Text variant="text-12">{polygon.name}</Text>
+                  <Text variant="text-12" className="opacity-50">
+                    {polygon.status}
+                  </Text>
+                </div>
+              </div>
+              <Icon
+                name={polygon.isUploaded ? IconNames.CHECK_POLYGON : IconNames.ELLIPSE_POLYGON}
+                className={classNames("h-6 w-6", { "animate-spin": !polygon.isUploaded })}
+              />
+            </div>
+          ))}
+        </div> */}
+      </ModalAdd>
+    );
+  };
+  const reloadSiteData = () => {
+    refetch();
+  };
+  const openFormModalHandlerConfirm = () => {
+    openModal(
+      <ModalConfirm
+        title={"Confirm Polygon Approval"}
+        content={contentForApproval}
+        commentArea
+        onClose={closeModal}
+        onConfirm={() => {}}
+      />
+    );
+  };
+
+  const openFormModalHandlerUploadImages = () => {
+    openModal(
+      <ModalAdd
+        title="Upload Images"
+        variantFileInput={VARIANT_FILE_INPUT_MODAL_ADD_IMAGES}
+        descriptionInput="Drag and drop a geotagged or non-geotagged PNG, GIF or JPEG for your site Tannous/Brayton Road."
+        descriptionList={
+          <Text variant="text-12-bold" className="mt-9 ">
+            Uploaded Files
+          </Text>
+        }
+        onClose={closeModal}
+        content="Start by adding images for processing."
+        primaryButtonText="Save"
+        primaryButtonProps={{ className: "px-8 py-3", variant: "primary", onClick: closeModal }}
+      >
+        {/* Next div is only Mocked data delete this children later*/}
```

Comment:
Can we get rid of the mocked data before this goes into `staging`?

Reply:
@diego-morales-flores-1996 @dottyy

Reply:
done, deleted.